src/components/pages/Home/Home.tsx

```diff
@@ -1,7 +1,36 @@
+import SearchBar from '../../inputs/Search/SearchBar/SearchBar';
 import './styles.css';
+import { useState } from 'react';
 
 function Home() {
-  return <div className="home">Home</div>;
+  const [searchBarState, setSearchBarState] = useState('success');
+  const [searchState, setSearchState] = useState('');
+
+  function searchAction() {
+    if (searchState == 'success') {
+      setSearchBarState('error');
+    }
+    if (searchState == 'error') {
+      setSearchBarState('search');
+    }
+    if (searchState == 'search') {
+      setSearchBarState('success');
+    }
+  }
+
+  return (
+    <div className="home">
+      <SearchBar
+        buttonAction={searchAction}
+        placeholderText="Enter a name"
+        searchState={searchState}
+        searchBarState={searchBarState}
+        onChangeHandler={setSearchState}
+        headerText={''}
+        footerText={''}
```

Comment:
per other comment, let's remove the state management from the parent and keep it internal to the component. we can add flags to indicate if the parent wants to see various states (e.g. `highlightError={error}`)

```suggestion
        buttonAction={searchAction}
        placeholderText="Enter a name"
        headerText={'Find a domain name.'}
        footerText={'FIGMA TEXT HERE FOR CHARACTER DETAILS'}
```